client/src/components/PillIcon/PillIcon.css

```diff
@@ -0,0 +1,28 @@
+.pill-container {
+  height: 3.434vw;
+  width: 11.566vw;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border-radius: 0.152vw;
+  box-shadow: 0px 0.16vw 0.16vw rgba(0, 0, 0, 0.25);
```

Comment:
Can you change all your units to either `px` or `rem` - we only want to use `vw` and `vh` for large things relative to the view height or window